packages/round-manager/src/mocks/handlers.js

```diff
@@ -0,0 +1,42 @@
+import {graphql} from "msw"
+
+export const handlers = [
+  graphql.query("GetGrantApplications", (req, res, ctx) => {
+    return res(
+      ctx.data({
+        roundProjects: [
+          {
+            id: "1",
+            metaPtr: {
+              protocol: 1,
+              pointer: "some-ptr"
+            },
+            title: "Some Title 1",
+            description: "Some project description 1",
+            payoutAddress: "0xSomeAddress"
+          },
+          {
+            id: "2",
+            metaPtr: {
+              protocol: 1,
+              pointer: "some-ptr2"
+            },
+            title: "Some Title 2",
+            description: "Some project description 2",
+            payoutAddress: "0xSomeAddress2"
+          },
+          {
+            id: "3",
+            metaPtr: {
+              protocol: 1,
+              pointer: "some-ptr3"
```

Comment:
would it be better to use actual pointer values ?

Reply:
Thinking so, but it should all be mocked by MSW anyways. We put a story in the backlog to setup a better testing framework around rendering context-aware components. I'll make this something real for now, though.